src/temgymbasic/gui.py

```diff
@@ -1278,3 +1349,89 @@ def build(self) -> Self:
 
         self.box.setLayout(vbox)
         return self
+
+
+class BiprismGUI(ComponentGUIWrapper):
+    @property
+    def biprism(self) -> 'comp.Biprism':
+        return self.component
+
+    @Slot(float)
+    def set_deflection(self, val: float):
+        self.biprism.deflection = val
+        self.try_update()
+
+    @Slot(float)
+    def set_offset(self, val: float):
+        self.biprism.offset = val
+        self.try_update(geom=True)
+
+    @Slot(float)
+    def set_rotation(self, val: float):
+        self.biprism.rotation = val
+        self.try_update(geom=True)
+
+    def sync(self, block: bool = True):
+        blocker = self._get_blocker(block)
+        with blocker(self.deflection_slider):
+            self.deflection_slider.setValue(self.biprism.deflection)
+        with blocker(self.offset_slider):
+            self.offset_slider.setValue(self.biprism.offset)
+        with blocker(self.rotation_slider):
+            self.rotation_slider.setValue(self.biprism.rotation)
+
+    def build(self) -> Self:
+        deflection = self.biprism.deflection
+        offset = self.biprism.offset
+        rotation = self.biprism.rotation
+
+        vbox = QVBoxLayout()
+
+        hbox = QHBoxLayout()
+        common_args = dict(
+            vmin=-0.5, vmax=0.5, decimals=2,
+        )
+        self.deflection_slider, hbox = labelled_slider(
+            value=deflection, name="Deflection", **common_args
+        )
+        self.offset_slider, _ = labelled_slider(
+            offset, -0.25, 0.25,
+            name="X-Offset", insert_into=hbox, decimals=2,
+        )
+        self.rotation_slider, _ = labelled_slider(
+            rotation, -np.pi, np.pi,
+            name="Rotation", insert_into=hbox, decimals=2,
+        )
```

Comment:
In keeping with the rest of the GUI, rotations should be in degrees when user-facing. The internal `Biprism` rotation value should remain radians, though!